peer_mentoring/account_management/models.py

```diff
@@ -61,6 +61,7 @@ class FriendRequest(models.Model):
         UserProfile, on_delete=models.CASCADE, related_name="received_requests"
     )
     created_at = models.DateTimeField(auto_now_add=True)
+    accepted = models.BooleanField(default=False)
```

Comment:
to keep the history of requests = nice (although need to double-check if a re-request would work no @JAWD-001, maybe log an issue because that's more of an edge case anyways)